mkosi.build

```diff
@@ -35,6 +35,13 @@ done
 # Run commands as the user running mkosi.
 SU="setpriv --reuid=${MKOSI_UID:-0} --regid=${MKOSI_GID:-0} --clear-groups"
 
+if [ -f "$SRCDIR/blktests/Makefile" ]; then
+    cd "$SRCDIR/blktests"
+    $SU make -j "$(nproc)"
+    make install
+    cd "$SRCDIR"
+fi
```

Comment:
Please do the copy to build directory dance here as well, otherwise you'll run into problems again when building for different distributions. Check ltp or btrfs-progs for an example

Reply:
done